tests/zfs-tests/tests/functional/arc/dbufstats_001_pos.ksh

```diff
@@ -55,10 +55,13 @@ function testdbufstat # stat_name dbufstat_filter
 
         [[ -n "$2" ]] && filter="-F $2"
 
-        verify_eq \
-	    $(grep -w "$name" "$DBUFSTATS_FILE" | awk '{ print $3 }') \
-	    $(dbufstat.py -bxn -i "$DBUFS_FILE" "$filter" | wc -l) \
-	    "$name"
+	from_dbufstat=$(grep -w "$name" "$DBUFSTATS_FILE" | awk '{ print $3 }')
+	from_dbufs=$(dbufstat.py -bxn -i "$DBUFS_FILE" "$filter" | wc -l)
+
+	if [[ $from_dbufstat -ne $from_dbufs ]]; then
+		within_percent $from_dbufstat $from_dbufs 99 \
+		    || log_fail "Stat $name differed too much"
+	fi
```

Comment:
What if instead we added a `verify_tolerance X Y Z` helper which checked that value X was within +/- Z% of Y.  Then we could do something like `verify_tolerance $from_dbufstat $from_dbufs 1`.